.github/workflows/regression-report.yml

```diff
@@ -0,0 +1,255 @@
+#####################################################################################
+# GitHub Action to generate Checkstyle report.
+#
+# Workflow starts when:
+# 1) issue comment - created
+#
+# Requirements:
+# 1) secrets.AWS_ACCESS_KEY_ID - access key for AWS S3 service user
+# 2) secrets.AWS_SECRET_ACCESS_KEY - security access key for AWS S3 service user
+#
+# If you need to change bucket name or region, change AWS_REGION and AWS_BUCKET_NAME variables.
+# For another bucket, you will need to change the secrets.
+#####################################################################################
+name: regression-report
+env:
+  AWS_REGION: us-east-2
+  AWS_BUCKET_NAME: "checkstyle-diff-reports"
+  USER_LOGIN: ${{ github.event.issue.user.login }}
+
+on:
+  issue_comment:
+    types: [ created, edited ]
+
+permissions:
+  contents: read
+  pull-requests: write
+
+concurrency:
+  # run_id is guaranteed to be unique and present
+  group: ${{ github.run_id }}
+  cancel-in-progress: false
+
+jobs:
+  parse_comment:
+    if: startsWith(github.event.comment.body, 'Github, generate report for')
+    runs-on: ubuntu-latest
+    outputs:
+      check_name: ${{ steps.parse.outputs.check_name }}
+      example: ${{ steps.parse.outputs.example }}
```

Comment:
combine to single variable - `config_bundle_path` 

this action should not care about module name at all, it just can run anything from test-configs repo. User define in comment path bundle.

Reply:
we do not need this variables at all. just remove them

Reply:
done